openslides_backend/action/action_handler.py

```diff
@@ -75,16 +78,25 @@ def handle_request(self, payload: Payload, user_id: int) -> ActionResponse:
         # TODO: Start a loop here and retry parsing actions and writing to event
         # store for some time if event store sends ModelLocked Exception
 
-        # Parse actions and creates events
-        write_request_element, results = self.parse_actions(payload)
-
-        # Send events to datastore
-        if write_request_element:
-            try:
-                self.datastore.write(write_request_element)
-            except EventStoreException as exception:
-                raise ActionException(exception.message)
-
+        retried = 0
+        payload_copy = deepcopy(payload)
+        while True:
+            # Parse actions and creates events
+            write_request_element, results = self.parse_actions(payload)
+
+            # Send events to datastore
+            if write_request_element:
+                try:
+                    self.datastore.write(write_request_element)
+                except DatastoreException as exception:
```

Comment:
What is the difference between `DatastoreException` and `EventStoreException`?

Reply:
DatastoreException is thrown in case of Datastore Problems like locked fields/models. EventStoreException is defined and imported and caught once, but seems to be never raised up to now.

Reply:
Can you refactor this in another PR? I'll create an issue...